lte/gateway/python/magma/pipelined/tests/test_ipv6_solicitation.py

```diff
@@ -106,6 +106,7 @@ def setUpClass(cls, *_):
                 'clean_restart': True,
                 'virtual_mac': 'd6:34:bc:81:5d:40',
                 'enable_nat': True,
+                'classifier_controller_id': 5,
```

Comment:
Why is `classifier_controller_id` required if `ng_service_enabled` is not set?

Reply:
We should be defined all variable which are used in a code otherwise getting Exception error like as below:
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/vagrant/magma/lte/gateway/python/magma/pipelined/tests/app/start_pipelined.py", line 238, in start_ryu_apps
    services.extend(manager.instantiate_apps(**contexts))
  File "/home/vagrant/build/python/lib/python3.8/site-packages/ryu/base/app_manager.py", line 509, in instantiate_apps
    self._instantiate(app_name, cls, *args, **kwargs)
  File "/home/vagrant/build/python/lib/python3.8/site-packages/ryu/base/app_manager.py", line 495, in _instantiate
    app = cls(*args, **kwargs)
  File "/home/vagrant/magma/lte/gateway/python/magma/pipelined/app/ipv6_solicitation.py", line 78, in __init__
    self.config = self._get_config(kwargs['config'])
  File "/home/vagrant/magma/lte/gateway/python/magma/pipelined/app/ipv6_solicitation.py", line 98, in _get_config
    classifier_controller_id=config_dict['classifier_controller_id'],
KeyError: 'classifier_controller_id'
E